src/ServiceControl.Audit.AcceptanceTests/Setup/DirectoryDeleter.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace ServiceControl.Audit.AcceptanceTests.Setup
+{
+    using System.Diagnostics;
+    using System.IO;
+    using System.Linq;
+    using System.Security.AccessControl;
+    using System.Security.Principal;
+
+    static class DirectoryDeleter
```

Comment:
should this helper method also be used in other places where this approach is used? (e.g. https://github.com/Particular/ServiceControl/blob/c1a58beef3a405209c2664186da83e611574b614/src/ServiceControl.Audit.AcceptanceTests/TestSupport/ServiceControlComponentRunner.cs )

Reply:
Yeah, the code is exactly the same. I will push the change and ensure tests continue to pass.